### PR TITLE
Sleep tight ARDS, you won't be missed

### PR DESCRIPTION
**Pull request by Thunder12345**
:cl: Thunder12345
del: Acute Respiratory Distress Syndrome has been removed as a virology symptom
/:cl:

ARDS: The ultimate ggnore symptom. With the right combination of other symptoms, can paralyse the victim, do massive amounts of oxyloss instantly, and gives zero warning of its existence until you fall down dying.

Counterplay consists of praying to god someone can cure themselves before they start being constantly knocked out, then calling the shuttle anyway because everyone else is dead before a vaccine can be distributed. This is in no way balanced, and serves as a virtually guaranteed round ender the moment it is released.

### CODE REVIEW

**Comment by k-k-karma** [comment]
just use an epipen lmao

**Comment by imsxz** [comment]
hey thunder thats what all the level 7 lethal symptoms are. stronger versions of already existing symptoms. alkali perspiration is just spontaneous combustion but stronger, autonecrosis is just the other brute damage one but stronger, ARDS is just choking but stronger.

This PR is extremely ided, especially given that ARDS is the weakest of those 3 symptoms by far, since epinepheirne exists in its current form and makes it impossible to die to oxygen damage.

**Comment by kingoftheautism** [comment]
I agree with imsxz, this is clearly i ded PR, since LVL7 bad symptoms are stronger version of already existing stuff, epipens exists, you can scavenge them in normal medkits and oxyloss medkits, and basically you just mad because you got killed and didnt know how to counter it

**Comment by subject217** [comment]
I don't think ARDS is really that dangerous. Everyone spawns with epinephrine and you can easily get more along with salbutamol from all the oxygen kits placed on the station.

**Comment by Tlaltecuhtli** [comment]
everyone starts with an epi pen and i think i had it once and it was jokingly easy to counter as it took like 5 min to get you on crit, also if you consider this lethal u better not check the one synthom that gives u 60u heparin hahaahha

**Comment by Thunder12345** [comment]
Did you miss the part where if you get the right stats it paralyses you every time you get hit with the oxyloss? At least the other symptoms give you a chance to fight back, rather than spending much of your time stunlocked

**Comment by Pandolphina** [comment]
I remember the round where the "I ded" comes from.

Literally everyone died because of lack of gloves or biosuits, there were massive warnings about viruses and it sends chat messages the moment it starts doing minir damage.

Like just epipen and then lynch viro for the cure.

**Comment by Thunder12345** [comment]
The damage isn't minor if you hit both the threshholds, it's enough to reach the damage slowdown, as well as injecting you with paralysis chems, with no warning beforehand that this is going to happen.

**Comment by Adanps** [comment]
Virologist is my most played job, which helps me know a lot about the role and its many difficulties. At its simplest, if you want to kill people dead you're going to want to use one of the three lethal level 7 symptoms. That's not a lot of options, and taking out ARDS will just make you just see more of the other two honestly.

ARDS tends to be the weakest in terms of damage, usually killing slower than both Alkali Perspiration and Autophagocytosis Necrosis. It mostly makes up for this by stunning victims when the stat threshold for paralysis is met. That is supposed to be the niche of ARDS, slower death but more debilitation. However, unlike what you seem to think, it is still the easiest to deal with thanks to epinephrine which completely stops the damage, and everyone starts off with a convenient pen full of the stuff. IIRC, even if someone dies to the symptom, they can be revived quickly via defibrillation since it is oxygen damage. 

The way you phrase it in the PR, it sounds like ARDS just suddenly stunned and killed you without any warning (like most antags usually do). This is totally bullshit though because of the way viruses are balanced. All three lethal level 7 symptoms have a stage where they start dealing minor damage which reveals their presence before moving on to kill the victim. If the virus has good speed then this can be a short period of time, but this means less time for victims to spread the virus before they die and it also forces the virus maker to make the virus less stealthy, resistant, or infectious as a result. 

However, even this only happens in the case where all other noticeable symptoms are neutered so that they don't alert the crew early. Otherwise the first time someone sniffles and proceeds to yell "VIRUS" over the radio will give it away really fast. But if you neuter all your other symptoms you won't get their benefits, such as sneezing spreading the virus more or coughing temporarily stunning victims. In this case you're suddenly relying on one level 7 symptom and the stats to do all the work, which makes the virus less effective.

At its core, viruses with lethal level 7 symptoms cannot be both stealthy and infective. This is because they cannot have stats of 2 stealth and 10 transmission at the same time. 2 Stealth makes the virus invisible to medHUDs and medical scanners. 10 transmission is where viruses begin to start spreading fast, because victims touching others will spread the virus. If the virus doesn't have 10 transmission or higher it is fairly easy to contain the outbreak. If it is does not have 2 stealth or higher then the virus is discovered quickly and a cure is much easier to discover. The only way around this is a 2 stealth virus with sneezing, but even then everyone is sneezing and it's really obvious there is a virus.

By the way, did you know all those biohazard suits scattered around certain parts of the station make you nearly impossible to be infected when combined with internals? If you hear over the radio that there's a lethal virus being spread you might consider trying to get one. Also if you have internals equipped and don't touch others you're very unlikely to get infected. You can actually avoid infection if you're careful about it.

So let's say you know all this and you still get infected by a virus suddenly. Since, you've been infected the virus is probably not very stealthy and a doctor will be able to diagnose you and tell you the cure. If there's no doctors you can use a medical scanner to find it out yourself. Once they do you can all the cures except gold from chemistry or a vending machine (I'm not joking, vending machines save lives). Chug down a few units of that cure and after a few seconds you're no longer dying, great! 

Anyone who gets cured can take this further and make vaccines using their blood and a pandemic machine. You can find pandemics in virology or the holodeck medical, and pandemic boards to make your own machine in tech storage or R&D. Then you can distribute tons of vaccine bottles that instantly cure victims and make anyone else impossible to infect with the same virus. Clone or defib whoever died before they were cured and lynch the virologist so they never make another virus again. Problem solved.

What really bothers me is that very few people seem to realize that lethal viruses aren't nearly the unstoppable station killers they're made out to be. As a result I seem to keep seeing these "I ded" removal PRs for virus symptoms when they inevitably die to one. This is not great when virology is already a bit lacking in symptoms that make for unique, useful viruses. I am obviously biased but I seriously think that if people died to viruses more often they would actually learn to counter them instead of having a knee jerk response to remove them all instead.

TL;DR : PR wants to remove a symptom for doing what it is meant to do while exaggerating what viruses using the symptom can actually do. I make an overly long explanation of why ARDS is fine as it is, and viruses in general are not actually as unstoppable killers as a lot of people seem to think.

P.S. : Don't you dare fucking tell me ARDS isn't needed because choking exists. Choking has the worst stats of any symptom in the game, and it the oxy damage is so pathetic that you might actually just recover because of natural healing of oxy damage. Seriously, buff choking it's awful.

**Comment by Adanps** [comment]
Wow I really made that comment way too long.

**Comment by BeeSting12** [comment]
Did you write a whole eight paragraph essay on why this symptom should stay in the game on an i ded PR that will probably get closed anyway

**Comment by Adanps** [comment]
Yeah I'm a little embarrassed about it. That wasn't what I was planning to do, but it's what it ended up becoming. I really just want everyone to be more aware of the limitations of viruses so they can stop being angry about them.

**Comment by ExcessiveUseOfCobblestone** [comment]
Every item that has been nerfed/removed have had limitations to them. That doesn't make them fun/enjoyable to play against.

I'm not a fan of viro in it's current iteration and think it's long overdo for a rework that isn't just making cute new symptoms to pad stats/heal/kill.

**Comment by Steelpoint** [comment]
Main issue I have with the Virology balancing is that we made that huge mess of utterly reworking Viro to make the healing viruses far less effective (which in this age the only good healing virus I think of is Healing Coma which only works when you are in crit) only for us to just add Nanites which do the same damn thing old Viro did.

**Comment by XDTM** [comment]
I'm working on a grounds-up rework of virology as a whole, although i still have to decide how to handle some aspects of it. I'll post the design doc somewhere when it's complete.

**Comment by ExcessiveUseOfCobblestone** [comment]
@Steelpoint the actual problem is that's literally the only thing you can do as nonantag.

**Comment by ExcessiveUseOfCobblestone** [comment]
I was thinking readd beauty component and at a beauty threshold there's a chance you can get diseases (dirtier rooms are able to cause more deadlier diseases).

I wanted to readd and edit the beauty component anyways so maybe ill be bothered enough to work on it.
